docker/Dockerfile

```diff
@@ -0,0 +1,33 @@
+ARG arch=amd64
+FROM ${arch}/ruby:2.6.3-stretch
```

Comment:
This seems to be a problem. The glibc in Debian stretch is too new. And there is no old Debian available for official ruby dockers.

I will explore whether it is possible to use alpine and musl-libc to build the Ruby.

Reply:
Can `FROM debian/eol:wheezy` not be used? Why does it need to be a Ruby docker?

Reply:
> Can FROM debian/eol:wheezy not be used?

I will take a look. It seems that `debian/eol:wheezy` indeed supports multiarch. However, I am not sure how to pull docker image for a specific arch as they all share the same tag. It seems that I have to pull using the hash of the image.

> Why does it need to be a Ruby docker?

Because we need it to run Homebrew. This is specially important if there is not portable-ruby available. For example, currently we need an external ruby to run Homebrew on ARM as there is no portable-ruby 2.6.3 for ARM. Basically, this is a bootstrap problem. I was thinking to use Ruby docker to avoid it.

Reply:
> I will take a look. It seems that `debian/eol:wheezy` indeed supports multiarch. However, I am not sure how to pull docker image for a specific arch as they all share the same tag. It seems that I have to pull using the hash of the image.

I reckon use that image as-is for Linux and we can deviate for the ARM builds.

> Basically, this is a bootstrap problem. I was thinking to use Ruby docker to avoid it.

👍 seems like a good idea for ARM and we could consider for x86_64 if it's a problem after that?

Reply:
> I will take a look. It seems that debian/eol:wheezy indeed supports multiarch. However, I am not sure how to pull docker image for a specific arch as they all share the same tag. It seems that I have to pull using the hash of the image.

Does not it not pull the appropriate image for the current architecture by default?